### ISSUE TITLE
Xbox 360 usb joypad won't register the start key right.

### ISSUE THREAD

**Issue by Dechstreme** [visualboyadvance-m/visualboyadvance-m#394]
### Expected behavior
To be able to map the GBA start Key to the start key of the joypad
### Actual behavior
The emulator is identifying the start key of the joypad as the Esc button when you press it, making it close the emulator and prevent from mapping the start key to the emulator.
### Steps to reproduce the behavior
Press start on the joypad, it doesn't even have to have a ROM playing.
### Crash bugs

If this is a crash bug, and you have the time and inclination, please provide a
symblic backtrace.

Instructions for doing so are
[here](https://github.com/visualboyadvance-m/visualboyadvance-m/blob/master/README.md#reporting-crash-bugs).

### ROM details

Which games are affected by the issue (please state if GBA or GB game):
It affects the emulator itself, it doens't need to have a rom playing.
Which games, if any, are NOT affected by the issue (please state if GBA or GB game):
None, all games are affected the same way.
Include screenshots of ROM Information (File > ROM Information):
N/A
### Option details

Video driver (simple, opengl):

Sound driver (directsound, xaudio, openal, SDL):

### Build details

Windows 10 professional

### Hardware details:

Xbox 360 usb controller

**Comment by rkitover** [comment]
Are you using 2.1.1? Because in that version the esc key mapping has been removed.

**Comment by Dechstreme** [comment]
I’m supposed to have the latest. I don’t know why it’s doing it. I tried downloading it from your site but it had the exact same data size as the one I got from another site. It also had the same date.


Sent from Yahoo Mail for iPhone


On Monday, March 25, 2019, 7:11 PM, Rafael Kitover <notifications@github.com> wrote:


Are you using 2.1.1? Because in that version the esc key mapping has been removed.
